docs/case_buttons.rst

```diff
@@ -1,5 +1,5 @@
-
-python3-pt-buttons Reference
+=============================
+pi-top [4] Case Buttons
 =============================
 
 This library allows you to interact with the function button on the pi-top [4]. You can write code to respond to these buttons being pressed and released. The following code example shows two different ways of doing this.
```

Comment:
```suggestion
This module allows you to interact with the buttons on the front of a pi-top [4]. You can write code to respond to these buttons being pressed and released. The following code example shows two different ways of doing this.
```

Reply:
We probably want to think about a template for all of these pages. I guess they would all be called modules?

Reply:
yeah, modules is the right call on these pages...